motion/los_guidance/scripts/los_guidance_node.py

```diff
@@ -228,12 +254,7 @@ def lookaheadBasedSteering(self):
 
 class LosPathFollowing(object):
     """
-    This is the ROS wrapper class for the LOS class.
-    Attributes:
-            _feedback	A vortex_msgs action that contains the distance to goal
-            _result		A vortex_msgs action, true if a goal is set within the
-                                    sphereof acceptance, false if not
-
+        self.reference_mod
     Nodes created:
```

Comment:
Why has this been deleted? Revert maybe, and this is crazy, but heres my number, so call me maybe